site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -481,3 +481,4 @@ FusionAuth Cloud has the same link:/docs/v1/tech/reference/limitations/[limitati
 * The IP addresses of a FusionAuth Cloud deployment are not fixed. Whenever possible, use the domain name, which is fixed. If you need IP addresses of the deployment nodes for purposes such as a network firewall allow list, please open a https://account.fusionauth.io/acccount/support/[support ticket], but be aware that they may change. Please upvote or comment on this https://github.com/FusionAuth/fusionauth-issues/issues/1393[open issue about static IP addresses in FusionAuth Cloud].
 * If you are on FusionAuth Cloud and you find that some requests are failing, it is possible you are being rate limited. This isn't intentional, but an automated part of our infrastructure to ensure FusionAuth Cloud performance and security. If you are rate limited but need these requests to occur, please open a https://account.fusionauth.io/acccount/support/[support ticket] with details.
 * If you want to run Advanced Threat Detection, an Enterprise feature, you'll need a cloud deployment with sufficient memory. Currently that means it must be a Large or X-Large size.
+* With HA deployments, requests to public metric endpoints such as `/api/status` are round-robined to individual servers and not aggregated across the deployment.
```

Comment:
Maybe .... So make sure that you check both nodes!

Reply:
No way for them to do so, unfortunately.

Reply:
We don't expose/document individual instances.